Mathlib/RingTheory/NonUnitalSubsemiring/Basic.lean

```diff
@@ -304,8 +304,7 @@ theorem coe_map (f : F) (s : NonUnitalSubsemiring R) : (s.map f : Set S) = f ''
 
 @[simp]
 theorem mem_map {f : F} {s : NonUnitalSubsemiring R} {y : S} : y ∈ s.map f ↔ ∃ x ∈ s, f x = y := by
```

Comment:
```suggestion
theorem mem_map {f : F} {s : NonUnitalSubsemiring R} {y : S} : y ∈ s.map f ↔ ∃ x ∈ s, f x = y := Iff.rfl
```
so that it can be used by `dsimp`